app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name, age):
```

Comment:
Missing type annotation for function argument 'name'
Missing type annotation for function argument 'age'